robot_smach_states/src/robot_smach_states/human_interaction/show_image.py

```diff
@@ -1,30 +1,30 @@
-from __future__ import absolute_import
-
-# ROS
-import os
 import rospy
 import smach
 
+from robot_smach_states.util.designators import check_type
 
-class ShowImageState(smach.State):
-    """
-    This state allows images to be shown on the hero display given the package in which the image is located and the
-    path of the image within that package
-    """
 
```

Comment:
Do keep the high level description of what the state does

Reply:
Oh that wasn't the plan

Reply:
I have put it back, but I have changed it too.